src/Assistants/PayoneAssistant.php

```diff
@@ -0,0 +1,625 @@
+                        + [
+                            $paymentCode.'AllowedDeliveryCountries' => [
+                                'type' => 'checkboxGroup',
+                                'defaultValue' => $this->getActiveCountriesValues(),
+                                'options' => [
+                                    'name' => 'Assistant.allowedDeliveryCountries',
+                                    'required' => true,
+                                    'checkboxValues' => $this->getDeliveryCountries4AmazonPay()
+                                ]
+                            ],
+                        ]
+                        + $this->getAuthorizationConfig($paymentCode) +
+                        [
+                            $paymentCode.'Sandbox' => [
+                                'type' => 'select',
+                                "defaultValue" => 1,
+                                'options' => [
+                                    'name' => 'Assistant.Sandbox',
+                                    'listBoxValues' => [
+                                        [
+                                            "caption" => 'Assistant.sandboxProductiveOption',
+                                            "value" => 1
+                                        ],
+                                        [
+                                            "caption" => 'Assistant.sandboxTestingOption',
+                                            "value" => 0
+                                        ]
+                                    ]
+                                ]
+                            ]
+                        ]
+                ]
+            ]
+        ];
+
+        return $config;
+    }
+
+
+    /**
+     * @param string $paymentCode
+     * @return array
+     */
+    protected function getMinMaxAmountConfig(string $paymentCode): array
+    {
+        return [
+            $paymentCode.'MinimumAmount' => [
+                'type' => 'double',
+                'defaultValue' => 0,
+                'options' => [
+                    'isPriceInput' => true,
+                    'decimalCount' => 2,
+                    'name' => 'Assistant.MinimumAmount'
+                ]
+            ],
+            $paymentCode.'MaximumAmount' => [
+                'type' => 'double',
+                'defaultValue' => 0,
+                'options' => [
+                    'isPriceInput' => true,
+                    'decimalCount' => 2,
+                    'name' => 'Assistant.MaximumAmount'
+                ]
+            ]
+        ];
+    }
+
+    /**
+     * @param string $paymentCode
+     * @return array
+     */
+    protected function getDeliveryCountriesConfig(string $paymentCode): array
+    {
+        return [
+            $paymentCode.'AllowedDeliveryCountries' => [
+                'type' => 'checkboxGroup',
+                'defaultValue' => $this->getActiveCountriesValues(),
```

Comment:
Sollte die Liste der activeCountries nicht eingeschränkt werden auf die möglichen Countrys ($this->getDeliveryCountries()). Ist in den meisten Fällen ja nur "DE","AT","CH".